.github/workflows/frontend-deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Frontend - Deploy ecrc-api
+on:
+  workflow_dispatch:
+    inputs:
+      source_image_tag:
+        required: true
+        description: What is the tag that is going to be promoted?
```

Comment:
Same question as above, is this the GitHub tag?

Reply:
Changed. It now says "image tag."